templates/mathlib_stats.html

```diff
@@ -6,20 +6,19 @@
       <h1>Mathlib statistics</h1>
 
     <h2>Declaration Counts</h2>
-  <div class="row">
-    <div class="col-md-4">
-      <h3>Definitions</h3>
-      <h4>{{ num_defns }}</h4>
-    </div>
-    <div class="col-md-4">
-      <h3>Theorems</h3>
-      <h4>{{ num_thms }}</h4>
-    </div>
-    <div class="col-md-4">
-      <h3>Meta declarations</h3>
-      <h4>{{ num_meta }}</h4>
-    </div>
-  </div>
+
+    <table class="table table-borderless mt-4" style="font-size: 1.3rem">
+        <tr>
+            <th>Definitions</th>
```

Comment:
These headers are now styled sans-serif in my browser, in contrast to the section headers. Was that intentional? 
![image](https://user-images.githubusercontent.com/4967469/95976839-5c91e000-0e18-11eb-8ef7-e2e0323bfb2a.png)

Reply:
Yes, I found it a bit confusing that there were three levels of big, bold, serif headings.  The sans serif rendering looks a bit off in chromium (that's the screenshot, right?), I could tweak the font size/weight a bit if that bothers you.

Reply:
It's Chrome, but yeah. Can't say I feel very strongly either way, but yeah, maybe a little tweaking would help. Do you think there's too much space between the headers and the numbers?

Reply:
Ok, I've reduced the font size and space between the rows a bit.  I've also added a bit of indentation, which really helps a lot.